src/logic/dictionaryUtils.ts

```diff
@@ -27,7 +27,7 @@ export function filterMatchesByPlayerCount(matches: Match[], playerCount: number
 /**
  * Given a collection of Matches, filters them to matches after a certain date.
  * Optionally, provide an end date as well.
- * @param matches
+ * @param matches Matches to consider. Only filters by date! Not validness
```

Comment:
You can remove the "Only filters by date! not validness" portion of this. 

If you are worried about confusion here, you can move this comment into the description above. Your "only filters by date..." is describing the function, not the parameter, so it should be in the JSDoc describing what the function is doing. 

Also, avoid the exclamation mark in JSDoc :D you can imagine that this is what other people will see in their intellisense, so this should be pretty "professional" :D

Reply:
if you want to call it out, do something like: 

NOTE: There are no assumptions made about the validity of these matches. 

or something like that.

Reply:
moved to description above.